docs/_static/nuts-registry.yaml

```diff
@@ -6,6 +6,72 @@ info:
   license:
     name: GPLv3
 paths:
+  /api/vendors:
+    post:
+      summary: "Adds a vendor to the registry"
+      operationId: "registerVendor"
+      tags:
+        - vendors
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Vendor'
+      responses:
+        '204':
+          description: "Vendor has been created successfully"
+        '400':
+          description: "incorrect data"
+  /api/vendor/{id}/claim:
```

Comment:
This seems to add the care organisation with a pub-key. What about the node with the vendor that should have the private key of the organisation?

Reply:
Discussed this: for now we assume that the node registering the organisation is the same that will be operating the organisation (registering and exposing endpoints) and thus in custody of the corresponding private key. We created #30 to improve the registration process and document it in detail.